tests/e2e/taqueria-plugin-ipfs-pinata.spec.ts

```diff
@@ -39,17 +39,20 @@ describe('e2e testing for the IPFS Pinata plugin with no JWT', () => {
 		} catch (_) {}
 	});
 
-	test('ipfs pinata plugin should show the correct help contents', async () => {
+	// Skipping due to help output changing
+	test.skip('ipfs pinata plugin should show the correct help contents', async () => {
```

Comment:
Why did we have to skip these?

Reply:
This was guidance from Michael K - if a test fails due to help output failing, we were advised to just skip the test. In this case, the help output changes because the --env, --build, and --version flags display in a different order than what they used to due to how/when they're loaded.